addons/website_profile/views/website_profile.xml

```diff
@@ -647,7 +647,7 @@
         </div>
         <div t-elif="validation_email_sent and not validation_email_done" t-att-class="done_alert_classes">
             <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
-            Verification Email sent to <i t-esc="user.email"></i>!<br/>
+            Verification Email sent to <t t-esc="user.email"/>!<br/>
```

Comment:
Note that this app is not handled by my team so in the future it's safer to wait for the related team to check this :wink: But I am sure they won't mind me removing an italic node, so let's go with it.

Although could you:
- Review the commit message to respect our conventions: max 72 characters per line
- Bonus: Take the opportunity to use t-out instead of t-esc

Thank you :+1:

Reply:
thanks for the review and suggestions @qsm-odoo .
updated